website/src/components/Features/Features.js

```diff
@@ -15,14 +15,14 @@ const FeatureList = [
         title: "Everything in one place",
         icon: require("../../../static/img/briefcase.png").default,
         description:
-          "The Taqueria CLI and VS Code plugin bring your development tools to you. Stop moving your code jumping between IDEs. With Taqueria’s plugin infrastructure you can use the tools you need all in one place.",
+          "The Taqueria CLI and VS Code plugin brings your development tools to you. Stop moving your code jumping between IDEs. With Taqueria’s plugin infrastructure you can use the tools you need all in one place.",
```

Comment:
'Taqueria CLI' and 'Vs Code Plugin' are both singular so this should be 'bring', not 'brings'